src/app/content/components/TableOfContents/keyboardSupport.hook.tsx

```diff
@@ -0,0 +1,171 @@
+import { HTMLElement, Element } from '@openstax/types/lib.dom';
+import { LinkedArchiveTreeNode } from '../../types';
+import React from 'react';
+
+export interface KeyboardSupportProps {
+  event: React.KeyboardEvent<HTMLAnchorElement>;
+  item: LinkedArchiveTreeNode;
+  isOpen?: boolean;
+  onSelect: () => void;
+}
+
+const getTreeItems = () => {
+  const treeItems = Array.from(document?.querySelectorAll('a[role="treeitem"]') as ArrayLike<HTMLElement>);
```

Comment:
this will behave strangely if there is more than one tree on the page, you should probably select relative to the currently focused link